Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -345,25 +374,34 @@ def construct_slack_msg(triggering_workflow: str,
     title = triggering_workflow
 
     if pull_request:
-        pr_number = pull_request.data.get('number')
-        pr_title = pull_request.data.get('title')
+        pr_number = pull_request.data['number']
+        pr_title = replace_escape_characters(pull_request.data['title'])
```

Comment:
PR title might contain escape characters that are problematic to the slack notification